File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/BustTag.java
package com.devingotaswitch.rankings.domain.appsync.tags;

import com.devingotaswitch.utils.Constants;

import java.util.Arrays;
import java.util.HashSet;

public class BustTag extends Tag {
    public BustTag(int count) {
        super(count, Constants.BUST_TAG, Constants.BUST_TAG_REMOTE, new HashSet<>(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)));
    }
}

File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/BustTag.kt
package com.devingotaswitch.rankings.domain.appsync.tags

import com.devingotaswitch.utils.Constants
import java.util.*

class BustTag(count: Int) : Tag(count, Constants.BUST_TAG, Constants.BUST_TAG_REMOTE, HashSet(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)))